### ISSUE TITLE
Missing Path Value in Zoho CRM Endpoint Resulting in 400 Bad Request

### ISSUE THREAD

**Issue by chiragthakkar** [verbb/formie#1998]
### Describe the bug

The Zoho CRM integration endpoint does not include the required path value (`crm/v2`), resulting in a `400 Bad Request` error when attempting to retrieve fields from Zoho CRM.

## Missing Path Value in Zoho CRM Endpoint Resulting in 400 Bad Request

**Repository**: [verbb/formie](https://github.com/verbb/formie)
**Link**:Issue in method [_getModuleFields](https://github.com/verbb/formie/blob/184747e4c9674ba47a2dea5d93358a6d3f4ca0d7/src/integrations/crm/Zoho.php#L420)

**Bug Description**:
The Zoho CRM integration endpoint does not include the required path value (`crm/v2`), resulting in a `400 Bad Request` error when attempting to retrieve fields from Zoho CRM.

**Steps to Reproduce**:
1. Generate a token using the provided method:
    ```php
    $this->client->post('https://accounts.zoho.com.au/oauth/v2/token', [
        'form_params' => [
            'client_id' => $this->clientId,
            'client_secret' => $this->clientSecret,
            'grant_type' => 'refresh_token',
            'refresh_token' => $this->refreshToken,
        ],
    ]);
    ```
    This correctly includes the path value `crm/v2`.

2. Attempt to retrieve fields using the endpoint:
    ```php
    $response = $this->client->request('GET', 'https://www.zohoapis.com.au/settings/fields?module=Leads', [
        'headers' => $this->getHeaders($token),
    ]);
    ```
    The path value `crm/v2` is missing here.

**Expected Behavior**:
The endpoint should include the path value `crm/v2`, resulting in a successful request:
    ```php
    $response = $this->client->request('GET', 'https://www.zohoapis.com.au/crm/v2/settings/fields?module=Leads', [
        'headers' => $this->getHeaders($token),
    ]);
    ```

**Actual Behavior**:
The endpoint does not include the path value `crm/v2`, resulting in a `400 Bad Request` error:

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

5.2.5

### Plugin version

3.0.0-beta.16

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.20"`.

**Comment by engram-design** [comment]
Fixed in [3.0.0](https://github.com/verbb/formie/releases/tag/3.0.0)
